metricbeat-yml-script.py

```diff
@@ -129,4 +113,4 @@ def parse_entry(entry):
 _add_modules()
 _add_shipping_data()
 
-os.system("metricbeat -e")
+os.system("metricbeat -e -c metricbeat.yml")
```

Comment:
Why we need this addition? we're in ${LOGZIO_DIR_PATH}

Reply:
we don't